### PR TITLE
Kafka low level components removal

### PR DESCRIPTION
**Pull request by arkadius**
## Checklist before merge
- [ ] Related issue ID is placed at the beginning of PR title in \[brackets\] (can be GH issue or Nu Jira issue)
- [ ] Code is cleaned from temporary changes and commented out lines
- [ ] Parts of the code that are not easy to understand are documented in the code
- [ ] Changes are covered by automated tests
- [ ] Showcase in dev-application.conf added to demonstrate the feature
- [ ] Documentation added or updated
- [ ] Added entry in _Changelog.md_ describing the change from the perspective of a public distribution user
- [ ] Added _MigrationGuide.md_ entry in the appropriate subcategory if introducing a breaking change
- [ ] Verify that PR will be squashed during merge

### CODE REVIEW

File: engine/flink/components/kafka/src/main/scala/pl/touk/nussknacker/engine/flink/util/transformer/FlinkKafkaComponentProvider.scala [hunk 1]
```diff
@@ -39,78 +28,11 @@ class FlinkKafkaComponentProvider extends ComponentProvider {
     val overriddenDependencies = TemporaryKafkaConfigMapping.prepareDependencies(config, dependencies)
     val docsConfig: DocsConfig = DocsConfig(config)
     import docsConfig._
-    val avro                    = "DataSourcesAndSinks#schema-registry--avro-serialization"
-    def universal(typ: String)  = s"DataSourcesAndSinks#kafka-$typ"
-    val schemaRegistryTypedJson = "DataSourcesAndSinks#schema-registry--json-serialization"
-    val noTypeInfo              = "DataSourcesAndSinks#no-type-information--json-serialization"
+    def universal(typ: String) = s"DataSourcesAndSinks#kafka-$typ"
```
Comment (pielas) [inline engine/flink/components/kafka/src/main/scala/pl/touk/nussknacker/engine/flink/util/transformer/FlinkKafkaComponentProvider.scala:31]:
```suggestion
    def universal(processingType: String) = s"DataSourcesAndSinks#kafka-$processingType"
```

File: engine/lite/components/kafka/src/main/scala/pl/touk/nussknacker/engine/lite/components/LiteKafkaComponentProvider.scala [hunk 2]
```diff
@@ -54,95 +29,11 @@ class LiteKafkaComponentProvider(schemaRegistryClientFactory: SchemaRegistryClie
   override def create(config: Config, dependencies: ProcessObjectDependencies): List[ComponentDefinition] = {
     val docsConfig = DocsConfig(config)
     import docsConfig._
-    val avro                    = "DataSourcesAndSinks#schema-registry--avro-serialization"
-    def universal(typ: String)  = s"DataSourcesAndSinks#kafka-$typ"
-    val schemaRegistryTypedJson = "DataSourcesAndSinks#schema-registry--json-serialization"
-    val noTypeInfo              = "DataSourcesAndSinks#no-type-information--json-serialization"
+    def universal(typ: String) = s"DataSourcesAndSinks#kafka-$typ"
```
Comment (pielas) [inline engine/lite/components/kafka/src/main/scala/pl/touk/nussknacker/engine/lite/components/LiteKafkaComponentProvider.scala:32]:
```suggestion
    def universal(processingType: String) = s"DataSourcesAndSinks#kafka-$processingType"
```

**Review by pielas — CHANGES_REQUESTED** [review]
Looks good, I only added two `typ` -> `processingType` replaces

**Comment by pielas** [comment]
Oh, I also missed that you didn't add any related tracking ID from this point of checklist `Related issue ID is placed at the beginning of PR title in [brackets] (can be GH issue or Nu Jira issue)`

File: engine/flink/components/kafka/src/main/scala/pl/touk/nussknacker/engine/flink/util/transformer/FlinkKafkaComponentProvider.scala (hunk 1 above)
Comment (arkadius) [reply engine/flink/components/kafka/src/main/scala/pl/touk/nussknacker/engine/flink/util/transformer/FlinkKafkaComponentProvider.scala]: It isn't a processing type - it's a component type. I changed it.

**Review by pielas — APPROVED** [review]
LGTM!

**Pull request merged**